pkgs/tools/wayland/hyprland-iio/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub, meson, cmake, pkg-config, dbus, ninja, ... }:
+
+stdenv.mkDerivation {
+  pname = "iio hyprland";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "short-circuit";
```

```suggestion
    owner = "JeanSchoeller";
```
Prefer pulling the main repo and using `fetchpatch` for your changes